libsel4vm/src/arch/arm/vgic/vgic.c

```diff
@@ -252,33 +252,6 @@ static inline bool is_enabled(struct gic_dist_map *gic_dist, int irq, int vcpu_i
     return is_spi_enabled(gic_dist, irq);
 }
 
-static inline void set_sgi_ppi_active(struct gic_dist_map *gic_dist, int irq, bool set_active, int vcpu_id)
```

Comment:
Yes, this seems unused, we are only accessing the field directly. I was a assuming this was kept in case a VMM needs to be able to manage something here. But activating interrupts unter the hood of a VM